apps/guide/src/content/02-getting-started/03-adding-your-bot-to-a-server.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Adding your bot to a server
+category: Getting started
+---
+
+# Adding your bot to a server
+
+After you [set up an application](./setting-up-an-application), you'll notice it's not in any servers yet. So, how does that work?
+
+Before you're able to see your bot in a server, you will need to add it by using a unique invite link.
```

Comment:
Bot invites are not unique?

Reply:
Word-to-word from the current guide—I think this means each invite link is specific to the bot (because of the application id), so it's unique in that sense?

Reply:
I find that specification (unique) to not add much, if any, value and to be more confusing than necessary

Reply:
Removed the word.